Gems/Atom/RHI/Vulkan/Code/Source/Platform/Windows/RHI/NsightAftermathHelpers.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include "GFSDK_Aftermath.h"
+#include "GFSDK_Aftermath_GpuCrashDump.h"
+#include "GFSDK_Aftermath_GpuCrashDumpDecoding.h"
+#include <AzCore/Debug/Trace.h>
+
+
+inline void AssertOnError(GFSDK_Aftermath_Result result)
+{
+    if(!GFSDK_Aftermath_SUCCEED(result))                                                              
+    {
+        AZStd::string errorMessage;
+        switch (result)
+        {
+        case GFSDK_Aftermath_Result_FAIL_DriverVersionNotSupported:
+        {
+            errorMessage = "Unsupported driver version - requires at least an NVIDIA R435 display driver.";
+            break;
+        }
+        case GFSDK_Aftermath_Result_FAIL_D3dDllInterceptionNotSupported:
+        {
+            errorMessage = "Aftermath is incompatible with D3D API interception, such as PIX or Nsight Graphics.";
+            break;
+        }
```

Comment:
This part probably isn't necessary on Vulkan. But since the plan is to refactor to share code later, it might not matter.